### PR TITLE
Dev/search choice page

### PR DESCRIPTION
**Pull request by SvetlanaBeynik**
## One Line Description
Added style for search-choice page

## Requirements

_What this code should do_

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome
image

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaSolovykh — APPROVED** [review]
Lgtm!

File: src/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { cardAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(cardAnatomy.keys);
+
+const baseStyle = definePartsStyle({
+  container: {
+    backgroundColor: "#f4f4eb",
+    borderColor: "#d7da5e",
```
Comment (rebekahcallkacz) [inline src/Card.tsx:10]: NON-BLOCKING: Is there a reason you didn't use the semantic tokens for these? I'd use `brandGray` and `green` if possible.

**Review by rebekahcallkacz — APPROVED** [review]
LGTM. We might want to consider a folder called `theme` where we put `CustomTheme.tsx`, `Card.tsx` and any other custom styles we create.

File: src/Card.tsx (hunk 1 above)
Comment (SvetlanaBeynik) [reply src/Card.tsx:10]: No, there is no reason, I will update it next time when I work on something else.

**Pull request merged**